gene_hints/citations/enrich_citations.py

```diff
@@ -74,9 +74,10 @@ def sort_genes(gene_dict, key):
 def pretty_print_table(raw_rows, num_rows):
     """Print rows with left-justified columns, for easy reading
     """
-    print(f"First {num_rows} rows:")
-    rows = raw_rows[:num_rows]
+    print(f"First {num_rows} rows")
+    rows = raw_rows[1: num_rows] # Offset accounts for meta-information
 
+    print(raw_rows[0][0])
```

Comment:
is this printing the column headers? / why print this after printing the rows?

Reply:
Good question!  This actually prints before printing the rows, but after printing the _number_ of rows.

It prints "meta-information", described more [here](https://github.com/broadinstitute/gene-hints/pull/21).  In brief, meta-information is data that applies to the whole file, like the number of days in the timeframe we're using to. analyze citations.  

So instead of redundantly listing something like `cite_days` for every row, it's declared once at the top of the file as a kind of special field.  That doesn't have the same number of columns as the main part of the file, which is why this line of code exists.